pubspec.yaml

```diff
@@ -26,6 +26,8 @@ dependencies:
   bloc_test: ^4.0.0
   url_launcher: ^5.4.2
   package_info: ^0.4.0+16
+  tutorial_coach_mark: ^0.3.0
```

Comment:
we also have to update our licenses now ;)

Reply:
True 😅

Reply:
done